Doc/library/sqlite3.rst

```diff
@@ -379,40 +379,44 @@ Module functions
    By using :func:`Binary`, developers can ensure robust handling of binary data in SQLite,
    preventing data corruption and enhancing application reliability.
 
-   **Examples:**
-
-   Basic example of storing and retrieving binary data:
+   **Basic example of storing and retrieving binary data:**
```

Comment:
If you want to use a title-like header, use `rubric`:

```suggestion
   .. rubric:: Basic example of storing and retrieving binary data
```

or simply write "The following code snippet is a basic example of storing and retrieving binary data:"